test/util/testnode/config.go

```diff
@@ -139,7 +139,7 @@ func DefaultConfig() *Config {
 func DefaultConsensusParams() *tmproto.ConsensusParams {
 	cparams := types.DefaultConsensusParams()
 	cparams.Block.TimeIotaMs = 1
-	cparams.Block.MaxBytes = appconsts.DefaultMaxBytes
+	cparams.Block.MaxBytes = int64(appconsts.DefaultUpperBoundMaxBytes)
```

Comment:
[question] why change the default? AFAICT the value used on mainnet is `1974272`. Ref: 

![Screenshot 2024-08-08 at 12 59 54 PM](https://github.com/user-attachments/assets/64f56a4a-16b0-4a5b-a0d0-b8bd98f93565)

```
DefaultMaxBytes= 1974272
DefaultUpperBoundMaxBytes=7897088
```

and the testnode should by default use those consensus params. With the exception of the max square size integration test, when do we need to create blocks bigger?

Reply:
I think I mistook this for the mempool MaxTxsBytes. This should stay the same